features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/extensions/store/app/am_store/app.js

```diff
@@ -24,6 +24,10 @@ app.server = function(ctx) {
                 url: 'widget',
                 path: 'widget.jag',
                 secured:false
+            }, {
+                url: 'applications',
+                path: 'applications.jag',
+                secured:false
```

Comment:
boolean should be true. This code indicates whether a user should be authenticated prior to been allowed to access the Page or API.If the property is not specified then the endpoint is assumed to be publicly accessible.